src/agora/txrx/txrx.h

```diff
@@ -27,7 +27,32 @@
 
 #if defined(USE_DPDK)
 #include "dpdk_transport.h"
-#endif
+
+#define USE_DPDK_MEMORY
```

Comment:
Maybe it is better to comment out this line. In my test, freeing DPDK buffer in dofft causes weird slow down and I am still investigating this issue.

Reply:
Remove the #define and added a comment to explain the error